integration/vm_factory/vm_cache_test.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+source "${cidir}/../../metrics/lib/common.bash"
+
+kata_runtime=$(command -v kata-runtime)
+
+# Environment variables
+KATA_CACHE_INIT="${kata_runtime} factory init"
+KATA_CACHE_STATUS="${kata_runtime} factory status"
+KATA_CACHE_DESTROY="${kata_runtime} factory destroy"
+KATA_CACHE_SOCK="/var/run/kata-containers/cache.sock"
+
+IMAGE="${IMAGE:-busybox}"
+TEST_PLAYLOAD="2"
+CONTAINER_NAME=$(random_name)
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+
+setup() {
+        extract_kata_env
+}
+
+echo "Starting vm cache test"
+setup
```

Comment:
Not a blocker" but since the rest of this test is so well structured, why not move this into a main function at the end of the script?

```sh
main()
{
    trap teardown EXIT

    echo "Starting vm cache test"

    setup

    echo "Running vm cache test"
    test_vm_cache_factory_init_destroy
}

main "$@"
```

Reply:
okay~~~ update asap. ;)